src/models/Episode.ts

```diff
@@ -20,6 +20,7 @@ export interface IEpisodeInput {
   isPublished: boolean;
   alternatives?: [PlayableContent];
   questionTime?: number;
+  duration: number;
```

Comment:
```suggestion
  duration?: number;
```
Could you update this so the `duration` field be optional on the `IEpisodeInput` too?
I believe we wont store the `duration` if the episode is Playable.